tests/python/gpu/test_operator_gpu.py

```diff
@@ -1478,6 +1479,48 @@ def test_cross_device_autograd():
 
     assert_almost_equal(dx, x.grad.asnumpy())
 
+
+# Running tests that may throw exceptions out of worker threads will stop CI testing
+# if not run in a separate process (with its own address space for CUDA compatibility).
+def can_fork_exec_process():
+    try:
+        if mp.get_start_method(allow_none=True) != 'spawn':
+            mp.set_start_method('spawn')
+        return True
+    except:
+        return False
+
+# The following 2 functions launch 0-thread kernels, an error that should be caught and signaled.
+def kernel_error_check_imperative():
+    os.environ['MXNET_ENGINE_TYPE'] = 'NaiveEngine'
+    a = mx.nd.array([1,2,3],ctx=mx.gpu(0))
+    b = mx.nd.array([],ctx=mx.gpu(0))
+    c = (a / b).asnumpy()
+
+def kernel_error_check_symbolic():
+    os.environ['MXNET_ENGINE_TYPE'] = 'NaiveEngine'
+    a = mx.sym.Variable('a')
+    b = mx.sym.Variable('b')
+    c = a / b
+    f = c.bind(mx.gpu(0), { 'a':mx.nd.array([1,2,3],ctx=mx.gpu(0)),
+                            'b':mx.nd.array([],ctx=mx.gpu(0))})
+    f.forward()
+    g = f.outputs[0].asnumpy()
+
+def test_kernel_error_checking():
+    # Execute test in a separate process with its own execution Engine.
+    if not can_fork_exec_process():
+        print('SKIP: needs process fork-exec support.')
```

Comment:
Under what circumstance would this not be supported?

Reply:
Python2 linux for one.  I was looking for a test that would be run at least once in the full CI environment.  I believe all Python3 environments should perform the test.